src/assets/css/homepage.scss

```diff
@@ -0,0 +1,72 @@
+.products-wrapper {
+    margin: 0;
+
+    @media screen and (min-width: $breakpoint-tablet){
+        display: grid;
+        grid-template-columns: auto auto;
+    }
+    @media screen and (min-width: $breakpoint-desktop){
+        display: grid;
+        grid-template-columns: auto auto auto;
+    }
+
+    .product {
+        display: flex;
+        font-family: 'Helveticaneuemedium';
```

Comment:
since `'Helveticaneuemedium'` is going to be used in several places it worths to create a variable for it.